src/sqlfluff/core/templaters.py

```diff
@@ -2,7 +2,10 @@
 
 import os.path
 import ast
-from typing import Dict
+from typing import Dict, Optional
+from dataclasses import dataclass

```

Comment:
This will break for users on python 3.6 unless I'm missing something?

Reply:
There's no `dataclasses` in setup.py

Reply:
Good catch, it must have been pulled in indirectly by another dependency, now I've added it to setup.py and requirements